src/pages/index.tsx

```diff
@@ -4,9 +4,18 @@ import Link from "next/link";
 import { Deck } from "@prisma/client";
 
 export default function Home() {
-  const { data, isLoading } = trpc.useQuery(["get-decks"], {
+  const userQuery = trpc.useQuery(["get-user"], {
     refetchOnWindowFocus: false,
   });
+  const { data, isLoading } = trpc.useQuery(
+    [
+      "get-decks-for-user",
+      { id: userQuery.data?.user?.id ? userQuery.data?.user?.id : "" },
```

Comment:
Get the user, and if we have a user pulled in, we fetch the decks for that user.
This line (13) is really ugly. Again, this flow will likely change with auth, but stacking queries is something to look into more deeply.